index.html

```diff
@@ -3,9 +3,18 @@
   <head>
     <meta charset="utf-8">
     <title>super-fantastic-dashboard</title>
+    <style>
+      body {
+        background-repeat: no-repeat;
+        background-position: center center;
+        background-attachment: fixed;
+        background-size: cover;
+      }
+    </style>
   </head>
   <body>
     <div id="app"></div>
     <script src="/dist/build.js"></script>
+    <script src="/static/js/background_image_loader.js"></script>
```

Comment:
That was my original plan but the idea to go this route was mentioned. I'm open to moving this functionality into Vue for sure.